### PR TITLE
Add unique name to action definition

### PR DESCRIPTION
**Pull request by tsurdilo**

### CODE REVIEW

**Comment by falko** [comment]
Why is this necessary and mandatory?

**Comment by tsurdilo** [comment]
Its a way to identify the action -- im working on a "mocking workflow data" bigger pr where its much needed. name is not a required param.

File: workflow/spec/spec.md [hunk 1]
```diff
@@ -692,6 +692,7 @@ instance in case it is an end state without performing any actions.
 
 | Parameter | Description | Type | Required |
 | --- | --- | --- | --- |
+| name | Unique action name | string | yes |
```
Comment (falko) [inline workflow/spec/spec.md]: Required should be `no`.

**Comment by tsurdilo** [comment]
nice catch . will fix

**Pull request merged**